CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
```

Comment:
Not sure about the catpialized scope in your commit message (though I realized this has been done for other commits in Marsha). If gitlint approves it then it should be fine :man_shrugging:

Reply:
There is no case control and I always use `AWS` :smile: do you think it's not good to use capitalized scope ?

Reply:
let's agree with gitlint ;-)